packages/fake-driver/lib/driver.js

```diff
@@ -6,15 +6,16 @@ import commands from './commands';
 
 class FakeDriver extends BaseDriver {
 
-  constructor () {
-    super();
+  constructor (opts = {}, shouldValidateCaps = true, driverArgs = {}) {
+    super(opts, shouldValidateCaps, driverArgs);
     this.appModel = null;
     this.curContext = 'NATIVE_APP';
     this.elMap = {};
     this.focusedElId = null;
     this.maxElId = 0;
     this.caps = {};
     this.fakeThing = null;
+    this.driverArgs = driverArgs;
```

Comment:
maybe we should add a TODO that we won't need this line once we're relying completely on basedriver from appium 2.0

Reply:
wait... I thought we _were_ completely relying on basedriver from appium 2.0?

Reply:
in general, drivers themselves are not yet restricted to use in appium 2.0 only. unless we want to fork each driver so that there is the 2.0 version of the driver and the 1.x version of the driver.